vendor/intel/avcstreamoutdemo.c

```diff
@@ -0,0 +1,1436 @@
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0}
+      },
+      RefPicList1:{
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0},
+          {4294967295, 0, 1, 0, 0}
+      },
+      luma_log2_weight_denom:0,
+      chroma_log2_weight_denom:0,
+      luma_weight_l0_flag:0,
+      luma_weight_l0:{0},
+      luma_offset_l0:{0},
+      chroma_weight_l0_flag:0,
+      chroma_weight_l0:{{0, 0}},
+      chroma_offset_l0:{{0, 0}},
+      luma_weight_l1_flag:0,
+      luma_weight_l1:{0},
+      luma_offset_l1:{0},
+      chroma_weight_l1_flag:0,
+      chroma_weight_l1:{{0, 0}},
+      chroma_offset_l1:{{0, 0}}
+  },
+  {
+      slice_data_size:1133,
+      slice_data_offset:1101,
+      slice_data_flag:0,
+      slice_data_bit_offset:32,
+      first_mb_in_slice:22,
+      slice_type:2,
+      direct_spatial_mv_pred_flag:0,
+      num_ref_idx_l0_active_minus1:0,
+      num_ref_idx_l1_active_minus1:0,
+      cabac_init_idc:0,
+      slice_qp_delta:-1,
+      disable_deblocking_filter_idc:0,
+      slice_alpha_c0_offset_div2:0,
+      slice_beta_offset_div2:0,
+      RefPicList0:{
+          {4294967295, 0, 1, 0, 0},
```

Comment:
it is 0xffffffff, invalidate picture, right?

Reply:
@XinfengZhang yes, it is